tests/group_vars/vagrant.yml

```diff
@@ -2,7 +2,7 @@
 # dataverse/tests/group_vars/vagrant.yml
 
 # un-nested so we can pass them at the CLI
-dataverse_branch: develop
+dataverse_branch: release
```

Comment:
just curious why this changed

Reply:
eh, testing different scenarios in different places. defaults/main.yml specifies release, jenkins develop; i suppose develop would be a better default for vagrant?

Reply:
ok, all good, just wondering, thanks! 🎉